### ISSUE TITLE
\n fails when no named queries are present

### ISSUE THREAD

**Issue by darikg** [dbcli/pgspecial#3]
```
test> \ns foo select 1;
Saved.
test> \n
+--------+----------+
| Name   | Query    |
|--------+----------|
| foo    | select 1 |
+--------+----------+
test> \nd foo
foo: Deleted
test> \n
Traceback (most recent call last):
  File "C:\Users\dg\Anaconda3\envs\pgcli2\Scripts\pgcli-script.py", line 9, in <module>
    load_entry_point('pgcli==0.19.2', 'console_scripts', 'pgcli')()
  File "C:\Users\dg\Anaconda3\envs\pgcli2\lib\site-packages\click\core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\dg\Anaconda3\envs\pgcli2\lib\site-packages\click\core.py", line 644, in main
    rv = self.invoke(ctx)
  File "C:\Users\dg\Anaconda3\envs\pgcli2\lib\site-packages\click\core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\dg\Anaconda3\envs\pgcli2\lib\site-packages\click\core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "c:\users\dg\documents\python\pgcli\pgcli\main.py", line 521, in cli
    pgcli.run_cli()
  File "c:\users\dg\documents\python\pgcli\pgcli\main.py", line 394, in run_cli
    click.echo_via_pager('\n'.join(output))
  File "C:\Users\dg\Anaconda3\envs\pgcli2\lib\site-packages\click\termui.py", line 203, in echo_via_pager
    return pager(text + '\n', color)
  File "C:\Users\dg\Anaconda3\envs\pgcli2\lib\site-packages\click\_termui_impl.py", line 262, in pager
    return _tempfilepager(text, os.environ['PAGER'], color)
  File "C:\Users\dg\Anaconda3\envs\pgcli2\lib\site-packages\click\_termui_impl.py", line 337, in _tempfilepager
    f.write(text.encode(encoding))
  File "C:\Users\dg\Anaconda3\envs\pgcli2\lib\encodings\cp437.py", line 12, in encode
    return codecs.charmap_encode(input,errors,encoding_map)
UnicodeEncodeError: 'charmap' codec can't encode characters in position 635-637: character maps to <undefined>
```

**Comment by amjith** [comment]
@darikg I'm not able to reproduce it. 

Can your setup display unicode characters? 

Can you try: 

```
select '日本語';
```

**Comment by darikg** [comment]
I think you're right. What unicode characters are being printed with `\n`?

**Comment by amjith** [comment]
I got a whee bit fancy with the help message that is printed with `\n`. Here's the output: 

```
hstore_test> \n
Named Queries are a way to save frequently used queries
with a short name. Think of them as favorites.
Examples:

    # Save a new named query.
    > \ns simple select * from abc where a is not Null;

    # List all named queries.
    > \n
    ╒════════╤═══════════════════════════════════════╕
    │ Name   │ Query                                 │
    ╞════════╪═══════════════════════════════════════╡
    │ simple │ SELECT * FROM abc where a is not NULL │
    ╘════════╧═══════════════════════════════════════╛

    # Run a named query.
    > \n simple
    ╒════════╤════════╕
    │ a      │ b      │
    ╞════════╪════════╡
    │ 日本語 │ 日本語 │
    ╘════════╧════════╛

    # Delete a named query.
    > \nd simple
    simple: Deleted

Time: 0.000s
hstore_test>
```

**Comment by amjith** [comment]
I'll be happy to remove that unicode.

I'm concerned that presence of unicode chars can screw with the operation in Windows.

**Comment by darikg** [comment]
Oh got it. My preference is just remove the unicode. It's kind of a bandaid solution but I'm not really able to debug any further because unicode is kind of a mystery to me and my impression is that Windows is really weird about it.

**Comment by darikg** [comment]
Fixed by #5
